src/rotest/cli/client.py

```diff
@@ -178,9 +180,15 @@ def main(*tests):
     if len(tests) == 0:
         tests = discover_tests_under_paths(options.paths)
 
+    # Filter tests by tags
+    tests = [test for test in tests if match_tags(get_tags(test),
```

Comment:
does it work?
i think the 'get_tags' function is supposed to get instances and not classes...